axum-macros/tests/debug_handler/fail/wrong_return_type.stderr

```diff
@@ -1,9 +1,10 @@
-error[E0277]: the trait bound `bool: IntoResponse` is not satisfied
+error[E0277]: the trait bound `bool: IntoResponseParts` is not satisfied
  --> tests/debug_handler/fail/wrong_return_type.rs:4:23
   |
 4 | async fn handler() -> bool {
-  |                       ^^^^ the trait `IntoResponse` is not implemented for `bool`
+  |                       ^^^^ the trait `IntoResponseParts` is not implemented for `bool`
   |
+  = note: required because of the requirements on the impl of `IntoResponse` for `bool`
```

Comment:
Its quite unfortunate that this error mentions `IntoResponseParts` at the top and not `IntoResponse`. I suppose rust is confused by the blanket `impl<T> IntoResponse for T where T: IntoResponseParts`.

Reply:
Yeah, same as with `IntoResponseHeaders` before. But why would we *need* the blanket impl? We would still accept more things with the remaining changes, without the blanket impl, right? And `Extension<T>` could independently implement `IntoResponse` so it's allowed as the last tuple element, or a "standalone" `impl IntoResponse` return value.

Reply:
Oh yeah of course! We don't need the blanket impl at all!